pgdrive/scene_creator/object/traffic_object.py

```diff
@@ -48,7 +48,6 @@ def __init__(self, lane, lane_index: LaneIndex, position: Sequence[float], headi
         self.lane_index = lane_index
         self.lane = lane
         self.body_node = None
-        self.name = random_string() if name is None else name
```

Comment:
So, the name is already initialized by the most basic Element object right?

Reply:
yes